src/command.c

```diff
@@ -453,6 +456,60 @@ int cmd_proxy(struct command *cmd, struct redis_data *data)
 
     if (strcasecmp(type, "INFO") == 0) {
         return cmd_proxy_info(cmd);
+    } else if (strcasecmp(type, "SETREMOTES") == 0) {
```

Comment:
`CONFIG SET NODE ip port ip port...` is easier to parse. And I suggest `NODE` should not be case sensitive.
What does `cluster_ok` mean?